lib/pointing_party_web/views/layout_helpers.ex

```diff
@@ -0,0 +1,10 @@
+defmodule PointingPartyWeb.LayoutHelpers do
+  use Phoenix.HTML
+
+  @doc """
+  Generates tag for inlined form input errors.
+  """
+  def signed_in?(conn) do
```

Comment:
The `?` here makes me thing this is expected to return a boolean, but the `conn.assigns[:username]` will return `nil` or a value.

What do you think about changing this to:

```elixir
def signed_in?(conn) do
  not is_nil(conn.assigns[:username])
end
```